pom.xml

```diff
@@ -579,6 +579,7 @@ under the License.
               <exclude>src/main/appended-resources/licenses/MIT-slf4j-api-1.7.30.txt</exclude>
               <exclude>src/main/appended-resources/licenses/EPL-1.0.txt</exclude>
               <exclude>src/main/appended-resources/licenses/unrecognized-javax.annotation-api-1.3.2.txt</exclude>
+              <exclude>.vscode/**</exclude>
```

Comment:
This should be added to https://creadur.apache.org/rat/apache-rat-plugin/rat-mojo.html, like `useEclipseDefaultExcludes` and others.